api/osb/osb_controller.go

```diff
@@ -162,13 +169,29 @@ func (c *Controller) proxy(r *web.Request, logger *logrus.Entry, broker *types.S
 	return resp, nil
 }
 
-func buildProxy(targetBrokerURL *url.URL, logger *logrus.Entry, broker *types.ServiceBroker) *httputil.ReverseProxy {
+func buildProxy(targetBrokerURL *url.URL, logger *logrus.Entry, broker *types.ServiceBroker) (*httputil.ReverseProxy, error) {
 	proxy := httputil.NewSingleHostReverseProxy(targetBrokerURL)
 	director := proxy.Director
 	proxy.Director = func(request *http.Request) {
 		director(request)
 		logger.Infof("Forwarded OSB request to service broker %s at %s", broker.Name, request.URL)
 	}
+
+	brokerClient, err := (&client.BrokerClient{}).New(broker, nil)
```

Comment:
It is more Go oriented to have a function `New` in the package which returns a new `BrokerClient`, which is configured

Reply:
10x will adjust

Reply:
Done